test/stablecoin_converter.js

```diff
@@ -948,6 +948,67 @@ contract("StablecoinConverter", async (accounts) => {
     await stablecoinConverter.submitSolution(batchIndex, owner2, orderIds2, volume2, prices2, tokenIdsForPrice2)
     assert.equal((await stablecoinConverter.currentPrices.call(2)).toNumber(), 0, "CurrentPrice were not adjusted correctly")
   })
+  it("checks that fees are granted correctly", async () => {
```

Comment:
nit: `it("grants fee surplus to solution submitter")`

Maybe also add a test case for the case where there is another solution overriding the first (making sure the original submitter doesn't get also get a reward)?

Reply:
This is exactly the second test, which is added in this PR, wasn't I?

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2FxT5LMH1Qi5BisdmMne%2Fgiphy.gif&token=imblind" data-gitmeme-token="imblind"><img src="https://media3.giphy.com/media/xT5LMH1Qi5BisdmMne/giphy.gif" title="Created by gitme.me with /imblind"/></a>